src/auth/axios.ts

```diff
@@ -22,7 +23,14 @@ const userAuthenticationExtraArgs = {
 
 const INVALID_ACCESS_TOKEN = 'Given access token is expired or invalid';
 
-const responseErrorInterceptor = (error: AxiosError) => {
+// TODO Use throughout application
```

Comment:
💭 Could you elaborate on this more? How will we go about adding this and changing old code?

Reply:
Good point, added elaboration in [this ticket](https://app.clickup.com/t/23k0qcf)